knowledge/responses/met_a_mean_vegan

```diff
@@ -0,0 +1,5 @@
+Just because someone who was vegan wasn’t very nice, it doesn't invalidate the entire movement and everyone who is a part of it. For instance, imagine someone was once shouted at by a ticket inspector on a train in Hungary because there was a misunderstanding about the ticket. Does that now mean we are morally justified to believe that all Hungarian men are not very nice? Or does that mean every ticket inspector on every train isn’t very nice? To believe that you can look down upon or judge on an entire demographic of people based on that one experience is obviously not logical in the slightest.
+
+Pooling the whole movement and its ideas on the behavior of some of its members is called the <a href="https://en.wikipedia.org/wiki/Ad_hominem">"Ad Hominem fallacy"</a>. It means that you dismiss the content of a message based on your opinion of the person bearing it. If a murderer spoke against racism, would that make being racist right?
+
+Also, people tend to get uncomfortable when their privileges and participation in oppression systems are mentioned to them. Men tend to get defensive when told they are being sexist, white people tend to get defensive when they are called on their racism, etc. It is very common for people who are comfortable with their privilege to think that those who called them out on it were rude, aggresive, or otherwise untoward. Vegans are speaking out for victims, and that means telling you that you are an active participant of a system of oppresion called <a herf="https://carnism.org/carnism/">carnism</a>. You perception of them being mean may just being your natural defense mechanisms of being called out on your privilege.
```

Comment:
Hey Valerie, Chau is caught up with work so I'm taking over the reviews for a little :).

Thanks for sending this in, I like this point you make here - which is "people naturally get defensive when told we're doing something wrong, but to ensure that we are always doing the best we can in this world, we need to carefully consider if we truly are doing something wrong instead of latching on to the instinctive feeling of not liking the messenger."

My concern here is that because you're using examples from sexism and racism, there are unfortunately large swaths of the population who do not believe these things are "real". And I worry that those people will check out or stop reading when they get to this part. If you could add some elements of the quoted text I have above _before_ this part, I think that would be great. Emphasize how important it is for them to carefully consider if they truly are doing harm before citing these examples. Let me know what you think.

The rest of the text is perfect, thanks for sending this in!